docs/source/initialsetup/mameintro.rst

```diff
@@ -1,43 +1,73 @@
+MAME is programmed in C++ and can currently emulate over 32000 individual
+systems from the last 5 decades.
 
 
 Purpose of MAME
-===============
+---------------
 
-The primary purpose of MAME is to preserve decades of arcade, computer, and console history. As technology continues to rush forward, MAME prevents these important “vintage” systems from being lost and forgotten.
+The primary purpose of MAME is to preserve decades of arcade, computer, and
+console history. As technology continues to rush forward, MAME prevents these
+important “vintage” systems from being lost and forgotten.
 
 
 Systems Emulated by MAME
-========================
+------------------------
 
 
-ProjectMESS contains a complete list of the systems currently emulated. As you will notice, being supported does not always mean that the status of the emulation is perfect. You may want
+The `Arcade Database <http://adb.arcadeitalia.net/>`_  contains a complete list
+of the systems currently emulated. As you will notice, being supported does not
+always mean that the status of the emulation is perfect. You may want
 
-1. to check the status of the emulation in the wiki pages of each system, accessible from the drivers page (e.g. for Apple Macintosh, from the page for the mac.c driver you can reach the pages for both **macplus** and **macse**),
-2. to read the corresponding **sysinfo.dat** entry in order to better understand which issues you may encounter while running a system in MAME (again, for Apple Macintosh Plus you have to check this entry).
+1. to check the status of the emulation in the wiki pages of each system,
+accessible from the drivers page (e.g. for Apple Macintosh, from the page for
+the mac.c driver you can reach the pages for both **macplus** and **macse**),
+2. to read the corresponding **sysinfo.dat** entry in order to better
+understand which issues you may encounter while running a system in MAME
+(again, for Apple Macintosh Plus you have to check this entry).
 
-Alternatively, you can simply see the status by yourself, launching the system emulation and taking a look to the red or yellow warning screen which appears before the emulation starts, if any. Notice that if you have information which can help to improve the emulation of a supported system, or if you can directly contribute fixes and/or addition to the current source, you can follow the instructions at the contact page or post to the MAME Forums at http://forum.mamedev.org/
+Alternatively, you can simply see the status by yourself, launching the system
+emulation and taking a look at the red or yellow warning screen which appears
+before the emulation starts, if any. Notice that if you have information which
+can help to improve the emulation of a supported system, or if you can directly
+contribute fixes and/or addition to the current source, you can do any of the
+following:
 
+    * Send in a pull request (for code) or open an issue (information) on our Github page.
```

Comment:
It’s styled as “GitHub” or “github” in all official material (we tend to use the former), never “Github”.

Reply:
Corrected. Also reformatted the text since I can now split lists.